src/components/explore/ProportionExplorerView.svelte

```diff
@@ -95,12 +92,37 @@ h2 {
 
     <div class=body-control-set>
       <label class=body-control-set--label>Categories</label>
+      {#if bucketOptions.length > maxVisibleCategories}
+        <OptionMenu multi on:selection={(evt) => {
+          dispatch('selection', { selection: evt.detail.keys, type: 'activeBuckets' });
+        }}>
+          {#each bucketOptions as bucketOption, i (bucketOption)}
```

Comment:
the ordering here in cases of a large quantity of options is not very usable. I have to go really far down the list to find the default checked ones, when what I think I want is the "top buckets" (the ones with actual color values mapped to them) on the top, then a divider line, then the rest below them (in whatever order). Is there a way to change this ordering so the most common ones are on the top, or the selected ones are on the top and in order?